examples/add-receiver-creator/rendered_manifests/configmap-agent.yaml

```diff
@@ -135,6 +135,10 @@ data:
         - system
         override: true
         timeout: 10s
+      transform/logs:
+        log_statements:
+        - context: log
+          statements: null
```

Comment:
To clarify my previous comment, this should not be rendered by default. Config should always have components that are used in the pipelines, otherwise it brings confusion for users and maintainers

Reply:
agree with your point.
I have updated the PR and the processor wouldn't be rendered by default.